Automattic/pocket-casts-android
# Add What's New screen for widgets

## Description

This PR adds "What's New" for widgets and removes stale code for old "What's New" features.

> ![note]
> I didn't update "What's New" version in the settings because at the moment I don't know which version widgets will target. I will update it during the integration with additional testing steps.

## Testing Instructions

1. Apply [this patch](https://github.com/Automattic/pocket-casts-android/files/14931241/whats_new.patch).
2. Open the app.
3. You should see widgets "What's New" screen.

## Screenshots or Screencast 

| Dark | Light |
| - | - |
| ![Screenshot_20240410-125455](https://github.com/Automattic/pocket-casts-android/assets/30936061/41ec6bc8-332b-4aff-a6f4-c60fb2e994e3) | ![Screenshot_20240410-125447](https://github.com/Automattic/pocket-casts-android/assets/30936061/b168f233-4832-4478-b1a5-bac5e0028261) |

## Checklist
 
#### I have tested any UI changes...
